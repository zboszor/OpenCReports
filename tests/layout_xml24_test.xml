<?xml version="1.0"?>
<!DOCTYPE OpenCReport SYSTEM "opencreport.dtd">
<OpenCReport>
	<Datasources>
		<Datasource name="myarray" type="'array'" />
	</Datasources>
	<Queries>
		<Query name="a" datasource="myarray" value="'array'" coltypes="'coltypes'" rows="0" cols="5" />
	</Queries>
	<Report fontSize="12">
		<NoData>
			<Output>
				<HorizontalLine size="1" color="'Black'" />
				<Line>
					<field value="'OpenCReports has links now!'" link="'https://github.com/zboszor/OpenCReports'" />
				</Line>
				<HorizontalLine size="1" color="'Black'" />
				<Line>
					<literal link="'https://github.com/zboszor/OpenCReports'" >OpenCReports has links now!</literal>
				</Line>
				<HorizontalLine size="1" color="'Black'" />
			</Output>
		</NoData>
	</Report>
</OpenCReport>
