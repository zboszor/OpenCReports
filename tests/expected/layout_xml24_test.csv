"OpenCReports has links now!"
"OpenCReports has links now!"
